cmd/dm-ctl/main.go

```diff
@@ -63,6 +66,40 @@ func main() {
 		os.Exit(2)
 	}
 
+	if len(cmdArgs) > 0 {
+		commandMode(cmdArgs)
+	} else {
+		interactionMode()
+	}
+}
+
+func collectArgs(args []string) []string {
```

Comment:
add any explanation for this function or for L35 ~ 43?

Reply:
add some explanation [here](https://github.com/pingcap/dm/pull/287/files#diff-61075abbd9aac2de49a1fe702632b302R38)